python/ipythonEmbed.py

```diff
@@ -57,15 +61,64 @@ def embed_kernel(module=None, local_ns=None, **kwargs):
 
         if app.poller is not None:
             app.poller.start()
+
         app.kernel.start()
         return app
 
+    def capture_output_streams():
+        sys.__stdout__, sys.__stderr__, sys.stdout, sys.stderr =  sys.stdout, sys.stderr, sys.__stdout__, sys.__stderr__
+
+    def release_output_streams():
+        sys.stdout, sys.stderr, sys.__stdout__, sys.__stderr__ =  sys.__stdout__, sys.__stderr__, sys.stdout, sys.stderr
+
+    def find_python_dir():
+        #We need to get the python directory like this, because
+        #sys.executable will return idaq.exe. This just goes two
+        #directories up from os.py location
+        return os.path.dirname(os.path.dirname(os.__file__))
+
+    def start_qtconsole():
+        try:
+            if kernel_app:
+                python_directory = find_python_dir()
+                cmd_line = [
+                    "{}/pythonw".format(python_directory),
+                    "{}/Scripts/jupyter-script.py".format(python_directory),
```

Comment:
Interesting, i seemed to just get that script through the normal install using conda. Regardless it might be better to just use something like `python -m qtconsole --existing kernel-8412.json`. Does that style of launching work on your installation?